mesh/mesh.hpp

```diff
@@ -896,6 +896,9 @@ class Mesh
        Return the face index of boundary element i. (3D) */
    int GetBdrElementEdgeIndex(int i) const;
 
+   // Checks if the info required for the above function is available.
+   bool BdrInfoAvailable() const;
+
```

Comment:
The function `GetBdrElementEdgeIndex` requires the boundary datamembers to be generated otherwise it will fail. The initialization of the NURBS mesh/space is unfortunalty a bit convoluted, resulting in the situation that the  `GetBdrElementEdgeIndex` could be called when they are not defined. This prompts the need for this check.

Reply:
I will update the comment. I am open for beter functionnames. Or another "mechanism" alltogether that does not rely on this check/data.